### PR TITLE
Add Python 3 compatibility classifier

### PR DESCRIPTION
**Pull request by pachewise**
Include metadata in setup.py so it shows this supports Python3. Also added keywords.

### CODE REVIEW

**Comment by pachewise** [comment]
Aligned with test matrix and kept Py 2.7 support (note this is just a label and does not affect whether this code actually supports 2.7). Feel free to remove that if that is the goal moving forward.

File: setup.py [hunk 1]
```diff
@@ -89,6 +89,16 @@ def initialize_options(self) -> None:
     packages=[
         'sqla_vertica_python',
     ],
+    keywords=['sqlalchemy', 'vertica', 'python'],
+    classifiers=[
+        'Development Status :: 3 - Alpha',
+        'License :: OSI Approved :: MIT License',
+        'Programming Language :: Python :: 2.7',
```
Comment (vinceatbluelabs) [inline setup.py]:
```suggestion
```

**Comment by vinceatbluelabs** [comment]
Hi @pachewise--thanks for the contribution!  Out of curiosity, does anything particularly automated read these, or do they mainly impact what appears on the PyPI site?

Yes, barring someone volunteering to support EOL versions of Python, let's leave 2.7 off.

**Comment by pachewise** [comment]
@vinceatbluelabs mostly PyPI (or your own personal index server), but then tools like Pyup read PyPI to determine Python3 support: https://github.com/pyupio/pyup/issues/269 (this library failing the pyup PY3 check is actually what prompted me to make this PR).

**Comment by vinceatbluelabs** [comment]
Makes sense!  Thanks again for the contribution!

**Pull request merged**